sqlmesh/dbt/builtin.py

```diff
@@ -26,6 +27,13 @@
 from sqlmesh.utils.errors import ConfigError, MacroEvalError
 from sqlmesh.utils.jinja import JinjaMacroRegistry, MacroReference, MacroReturnVal
 
+if t.TYPE_CHECKING:
+    from typing import Protocol
+
+    class Model(Protocol):
```

Comment:
Is this used for anything?

Reply:
Nope, leftover from previous iteration. Thanks!